src/Umbraco.Web.UI.Client/src/views/components/references/umb-tracked-references.html

```diff
@@ -47,6 +47,18 @@ <h5 class="mt0" style="margin-bottom: 20px;">
         </div>
     </div>
 
+    <div ng-if="vm.hasContentReferencesInDescendants">
+        <h5 class="mt0" style="margin-bottom: 20px;">
+            <localize key="references_labelUsedByDocuments">Used in Documents</localize>
+        </h5>
+
+        <umb-box>
+            <umb-box-content>
+                <localize key="references_labelUsedInContentDescendants">One of the underlying items is being used in content item</localize>
```

Comment:
I'm not sure that 'underlying' is the correct term here, at least to me it seems odd and not sure we use that term for descendants anywhere else? Could it be "One or more of this item's descendants is being used in another content item" is that more specific to what this is trying to tell the user?

Reply:
Update the translations